src/sim65/6502.c

```diff
@@ -215,7 +215,9 @@ int PrintCycles;
     ZPAddr = MemReadByte (Regs.PC+1);                           \
     Addr = MemReadZPWord (ZPAddr);                              \
     if (PAGE_CROSS (Addr, Regs.YR))                             \
+    {                                                           \
```

Comment:
Fixed that too. But that is what I mean: Where can I find this "style guide" or is there an automatic check where I can see when I break it? (I'm familiar with "Resharper" in this respect)
(Not that I find the style very consistent nor I would prefer it... ;) )
You making others really a hard time to contribute when even copy & paste of existing code leads to refinishing...